code/game/objects/items/devices/scanners.dm

```diff
@@ -537,6 +537,29 @@ GENE SCANNER
 			amount += inaccurate
 	return DisplayTimeText(max(1,amount))
 
+/proc/geigercounter_scan(mob/user, atom/A)

```

Comment:
this is a global proc, there's no `src` value that makes sense in this proc.

Did you mean to scope it to the scanner path?